src/main/resources/templates/jenkins-admin-settings.soy

```diff
@@ -114,6 +114,11 @@
 		{param checked: $server.altUrl /}
     	{param labelContent: 'Build Token Root Plugin (uses an alternate url for triggering builds)' /}
     {/call}
+    {call widget.aui.form.checkbox}
+        {param id:'jenkinsCSRF' /}
+        {param checked: $server.csrfEnabled /}
+        {param labelContent: 'Check this box if your Jenkins instance has the CSRF protection enabled' /}
```

Comment:
This is a pretty long message. I'd consider shortening it up a bit.

Reply:
As soon as the meaning is here, it's your code :)